src/ui/Facet/FacetSettings.ts

```diff
@@ -186,6 +186,10 @@ export class FacetSettings extends FacetSort {
         this.unselectItem(this.getSortItem(criteria.name));
       }
     });
+
+    if (this.sortSection) {
+      first(this.sortSection.sortItems).focus();
```

Comment:
Is it possible for the `sortSection` sortItems to be an empty array?

Reply:
I just checked and it doesn't seem like so. However, this shouldn't matter anymore since any first element with `tabindex >= 0` will be focused.